pkgs/tools/security/gnupg/24.nix

```diff
@@ -38,46 +78,54 @@ stdenv.mkDerivation rec {
-    '' + lib.optionalString (stdenv.hostPlatform.isLinux && withPcsc) ''
+  postPatch =
+    ''
+      sed -i 's,\(hkps\|https\)://keyserver.ubuntu.com,hkps://keys.openpgp.org,g' configure configure.ac doc/dirmngr.texi doc/gnupg.info-1
+    ''
+    + lib.optionalString (stdenv.hostPlatform.isLinux && withPcsc) ''
       sed -i 's,"libpcsclite\.so[^"]*","${lib.getLib pcsclite}/lib/libpcsclite.so",g' scd/scdaemon.c
     '';
 
-  configureFlags = [
-    "--sysconfdir=/etc"
-    "--with-libgpg-error-prefix=${libgpg-error.dev}"
-    "--with-libgcrypt-prefix=${libgcrypt.dev}"
-    "--with-libassuan-prefix=${libassuan.dev}"
-    "--with-ksba-prefix=${libksba.dev}"
-    "GPGRT_CONFIG=${lib.getDev libgpg-error}/bin/gpgrt-config"
-  ]
-  ++ lib.optional guiSupport "--with-pinentry-pgm=${pinentry}/${pinentry.binaryPath or "bin/pinentry"}"
-  ++ lib.optional withTpm2Tss "--with-tss=intel"
-  ++ lib.optional stdenv.hostPlatform.isDarwin "--disable-ccid-driver";
+  configureFlags =
+    [
+      "--sysconfdir=/etc"
+      "--with-libgpg-error-prefix=${libgpg-error.dev}"
+      "--with-libgcrypt-prefix=${libgcrypt.dev}"
+      "--with-libassuan-prefix=${libassuan.dev}"
+      "--with-ksba-prefix=${libksba.dev}"
+      "GPGRT_CONFIG=${lib.getDev libgpg-error}/bin/gpgrt-config"
+    ]
+    ++ lib.optional guiSupport "--with-pinentry-pgm=${pinentry}/${
+      pinentry.binaryPath or "bin/pinentry"
+    }"
+    ++ lib.optional withTpm2Tss "--with-tss=intel"
+    ++ lib.optional stdenv.hostPlatform.isDarwin "--disable-ccid-driver";
 
-  postInstall = if enableMinimal
-  then ''
-    rm -r $out/{libexec,sbin,share}
-    for f in $(find $out/bin -type f -not -name gpg)
-    do
-      rm $f
-    done
-  '' else ''
-    # add gpg2 symlink to make sure git does not break when signing commits
-    ln -s $out/bin/gpg $out/bin/gpg2
+  postInstall =
+    if enableMinimal then
+      ''
+        rm -r $out/{libexec,sbin,share}
+        for f in $(find $out/bin -type f -not -name gpg)
+        do
+          rm $f
+        done
+      ''
+    else
+      ''
+        # add gpg2 symlink to make sure git does not break when signing commits
+        ln -s $out/bin/gpg $out/bin/gpg2
 
-    # Make libexec tools available in PATH
-    for f in $out/libexec/; do
-      if [[ "$(basename $f)" == "gpg-wks-client" ]]; then continue; fi
-      ln -s $f $out/bin/$(basename $f)
-    done
+        # Make libexec tools available in PATH
+        for f in $out/libexec/; do
+          if [[ "$(basename $f)" == "gpg-wks-client" ]]; then continue; fi
+          ln -s $f $out/bin/$(basename $f)
+        done
 
-    for f in $out/libexec/; do
-      if [[ "$(basename $f)" == "gpg-wks-client" ]]; then continue; fi
-      ln -s $f $out/bin/$(basename $f)
-    done
-  '';
+        for f in $out/libexec/; do
+          if [[ "$(basename $f)" == "gpg-wks-client" ]]; then continue; fi
+          ln -s $f $out/bin/$(basename $f)
+        done
```

Comment:
Am I missing something or are the two identical for-loops to "[m]ake libexec tools available in PATH" and one can be deleted?
```suggestion
```

Reply:
I'm uncertain, but I'll test it without the second loop